pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -79,7 +80,7 @@
         continue;
       }
       GMT_ZONES.put(pgZoneName + Math.abs(i), timeZone);
-      GMT_ZONES.put(pgZoneName + NUMBERS[Math.abs(i)], timeZone);
+      GMT_ZONES.put(pgZoneName + Arrays.toString(NUMBERS[Math.abs(i)]), timeZone);
```

Comment:
Please use `new String(NUMBERS[Math.abs(i)])` here. It might deserve its own PR, however, I'm fine if you fix that here.
The idea is `Math.abs(i)` produces `7` while `new String(NUMBERS[Math.abs(i)])` would produce `07`